go.mod

```diff
@@ -54,6 +79,7 @@ require (
 )
 
 replace (
+	github.com/dapr/components-contrib => github.com/jjcollinge/components-contrib v1.0.0-rc1.0.20210325195722-6c8b0ef77d34
```

Comment:
Do not merge until this is reverted.

Reply:
@artursouza should I wait until the components-contrib PR is merged and then remove this replace and update the require to point to that version?

Reply:
Correct.